docs/usage.md

```diff
@@ -55,6 +55,76 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
+For something like fly, you would want the annotation matching the toplevel assembly (e.g. `Drosophila_melanogaster.BDGP6.46.{release}.gtf.gz` from ENSEMBL).
+As well as the comprehensive annotations for the primary and toplevel assemblies, and just the reference chromomes, GENCODE also provides "basic" annotations which only include representative transcripts, but we do not recommend using these.
+
+Gene annotations typically provide a primary identifier for each feature as well as a more common name.
+For example, the ENSEMBL ID `ENSG00000254647` corresponds to the `INS` gene which encodes the insulin protein.
+While the gene names may be more familiar and easier to understand it is important to retain and use the primary identifiers as the are unique for a given annotation and are much easier to map between annotation versions or sources.
+
+To take advantage of all the quality control modules implemented in the pipeline, the gene annotation should include a `gene_biotype` field which describes the function of each feature (protein coding, long non-coding etc.).
+This is usually the case for annotations from GENCODE or ENSEMBL but may not be if your annotation comes from another source.
+If your annotation does not include this field, please set the `--skip_biotype_qc` option to avoid running the steps that rely on it.
+
+:::note
+**GTF vs GFF**
+
+GFF (General Feature Format) is a tab-separated text file format for representing genomic annotations.
+GTF (General Transfer Format) is a specific implementation of this format corresponding to GFF version 2.
+The pipeline can accept both GFF and GTF but any GFF files will be converted to GFF so if a GTF is available for your annotation of choice it is better to provide that directly.
+
+More information and links to further resources are [available from ENSEMBL](https://www.ensembl.org/info/website/upload/gff.html).
+:::
+
+### Reference transcriptome
+
+As well as the reference genome sequence and annotation it is possible to provide a reference transcriptome FASTA file.
+These can be obtained from GENCODE or ENSEMBL but it is important to note that the sequences they provide only cover the reference chromosome and can result in inconsistencies if you have provided a primary or toplevel genome assembly and annotation.
+For this reason, we recommend to not provide a transcriptome FASTA and instead let the pipeline create it from the provided genome and annotation.
+As with the aligner indexes, it is possible to save the created transcriptome FASTA and BED files to a central location and provide it to future pipeline runs in order to avoid having multiple copies on your system but it is important to make sure that all genome, annotation, transcriptome and index versions match.
+
+### Indexes
+
+Creating the index files required for the alignment and/or pseudoalignment steps can be computationally intensive and the files they produce are quite large.
```

Comment:
There is already an 'indices' section that covers this. Could you suggest any improvements there please?